pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-circulation</artifactId>
   <groupId>org.folio</groupId>
-  <version>19.1.0</version>
+  <version>19.2.0</version>
```

Comment:
How was this version changed? Usually this is done with the maven release plugin, which does not appear to have been run.

Reply:
Reverted. Yes, plugin isn't still running.